recipes/connection_pool/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ConnectionPool" %}
+{% set author = "zhouyl" %}
+{% set version = "0.0.3" %}
+{% set sha256 = "3acb9bbd6ef9a6ca9d5e056c0afd14dbc5ca8d610d788b162412f6235a6e3759" %}
+
+package:
+  name: connection_pool
```

Comment:
Why add the underscore?

Reply:
It's the name of the import and the pip package